Rakefile

```diff
@@ -303,6 +327,7 @@ namespace :install do
   tasks << :blockly_symlink
   tasks << :apps if CDO.build_apps
   tasks << :shared if CDO.build_shared_js
+  tasks << :code_studio
```

Comment:
That does seem accurate, but we have it conditioned for our other parts. Maybe we create the symlink by default elsewhere, and only need to run this install step if we're replacing it with a symlink to built code?